core/templates/dev/head/services/PlaythroughService.js

```diff
@@ -44,9 +44,15 @@ oppia.factory('PlaythroughService', [
     var expStopwatch = null;
     var isLearnerInSamplePopulation = null;
 
-    var multipleIncorrectStateName = {};
+    var multipleIncorrectStateName = {
+        state_name: null,
```

Comment:
Deindent by 2.

/cc @oppia/dev-workflow-team -- this should be caught by the linter.

Reply:
The linter is disabled. I will enable the linter once tslint stuff is done. You can ignore the linter erros till then.